livekit/types.go

```diff
@@ -5,7 +5,7 @@ package livekit
 type TrackID string
 
 func StringsAsTrackIDs(trackIDs []string) []TrackID {
-	var asTrackID []TrackID
+	asTrackID := make([]TrackID, 0, len(trackIDs))
```

Comment:
Probably can make the same change in room names function at the bottom on this file to be consistent.

Reply:
Done